rest/data/project.go

```diff
@@ -344,14 +181,10 @@ func GetProjectEventLog(project string, before time.Time, n int) ([]restModel.AP
 		// don't return an error here to preserve existing behavior
 		return nil, nil
 	}
-	return getEventsById(id, before, n)
+	return GetEventsById(id, before, n)
 }
 
-func GetRepoEventLog(repoId string, before time.Time, n int) ([]restModel.APIProjectEvent, error) {
-	return getEventsById(repoId, before, n)
-}
-
-func getEventsById(id string, before time.Time, n int) ([]restModel.APIProjectEvent, error) {
+func GetEventsById(id string, before time.Time, n int) ([]restModel.APIProjectEvent, error) {
```

Comment:
Why was this made public?

Reply:
There was a public function wrapping this private function in 1 line, so it was essentially already public. I just removed the wrapper function and made this function public.

Reply:
I think it was actually worth keeping that function, even though it just wraps that one, because it was clear then that that one should be used for Repos and the other should be used for Projects, and this is just a logical helper function.

Reply:
That being said I think we should move both into model/project_events.go (and then ProjectEventsBefore can also be made private)